docs/overview/proof-of-burn.md

```diff
@@ -0,0 +1,36 @@
+# Proof-of-Burn
+
+Proof-of-Burn (PoB) is the consensus algorithm used by the Koinos blockchain. It shares properties with both Proof-of-Work (PoW) and Proof-of-Stake (PoS). At a high level, Proof-of-Burn emulates the behavior of Proof-of-Work while keeping the concept of real world assets such as mining rigs virtualized on-chain.
+
+### Similarities to Proof-of-Work
+Like Proof-of-Work, a potential block producer will acquire hash power by sacrificing capital in order to have higher chances of mining blocks. The more hash power one has, the more likely they will produce a block.
```

Comment:
Do we want to explain Proof of Work? The sacrifice of capital may not be readily apparent. There are costs to purchase mining rigs and costs to purchase electricity, both of which must be offset in order to be profitable.

When I talk with people about blockchain, most of the conversation is answering the question, "What exactly is Bitcoin mining?". There is not general knowledge about PoW.